apps/cookbook/src/app/page/side-nav/side-nav.component.ts

```diff
@@ -14,32 +16,58 @@ import { filter } from 'rxjs/operators';
 import { kebabToTitleCase } from '@kirbydesign/designsystem';
 
 import { routes } from '../../showcase/showcase.routes';
-import { navigationItems } from '../header/header.component';
 
 const KEY_DOWN = 'ArrowDown';
 
 interface SideNavLink {
-  path: string;
+  path?: string;
+  externalUrl?: string;
   name: string;
-  active: boolean;
 }
 
 @Component({
   selector: 'cookbook-side-nav',
   templateUrl: './side-nav.component.html',
   styleUrls: ['./side-nav.component.scss'],
 })
-export class SideNavComponent implements OnInit {
+export class SideNavComponent implements OnInit, AfterViewInit {
   private allShowcaseRoutes: SideNavLink[];
   filteredShowcaseRoutes: SideNavLink[][];
   filter: string = '';
 
   @Output() menuToggle = new EventEmitter<boolean>();
-  @Input() isMenuOpen = false;
+  @HostBinding('class.is-open')
+  @Input()
+  isMenuOpen = false;
+
+  sideNavGroups: { [key: string]: QueryList<ElementRef<HTMLAnchorElement>> };
 
   constructor(private router: Router) {}
 
-  navigationLinks = navigationItems;
+  ngAfterViewInit(): void {
+    this.sideNavGroups = {
+      resources: this.resourceLinks,
+      components: this.componentLinks,
+    };
+  }
+
+  navigationItems: SideNavLink[] = [
+    { name: 'Introduction', path: '/home/intro' },
+    { name: 'Guides', path: '/home/guides' },
+    {
+      name: 'Accessibility',
+      path: '/home/accessibility-in-kirby',
+    },
+    {
+      name: 'Extensions',
+      path: '/home/extensions',
+    },
+    {
+      name: 'Changelog',
+      externalUrl: 'https://github.com/kirbydesign/designsystem/releases',
+    },
+    { name: 'GitHub', externalUrl: 'https://github.com/kirbydesign/designsystem' },
+  ];
```

Comment:
Consider extracting to separate file as we do with app routes...

Reply:
Good point, changed it so we get the resource links from the app routes instead, similar to how we get showcase links: 094cee314